crates/interface/src/error.rs

```diff
@@ -128,6 +128,9 @@ pub enum InvalidTransaction {
     /// Other unrecoverable error.
     /// Unrecoverable means that tx cannot become valid in the future and should be dropped.
     OtherUnrecoverable(String),
+    FilteredByValidator {
+        deployer: Address,
```

Comment:
The field doesn't make sense anymore after error got renamed. I'd remove it or at least rename to `from`.

Reply:
fixed. there is one more question about `on_new_execution_frame` please check